src/tiled/preferences.cpp

```diff
@@ -84,6 +84,7 @@ Preferences::Preferences()
     mShowTilesetGrid = boolValue("ShowTilesetGrid", true);
     mLanguage = stringValue("Language");
     mUseOpenGL = boolValue("OpenGL");
+    mZoomWheelByDefault = boolValue("ZoomWheelByDefault");
```

Comment:
Did you miss that I called it `mWheelZoomsByDefault` and not `mZoomWheelByDefault`? I think the former makes more sense.